base/uk.ac.stfc.isis.ibex.scriptgenerator/src/uk/ac/stfc/isis/ibex/scriptgenerator/generation/ParametersConverter.java

```diff
@@ -0,0 +1,100 @@
+package uk.ac.stfc.isis.ibex.scriptgenerator.generation;
+
+import java.util.List;
+import java.util.Map;
+import com.google.gson.Gson;
+
+/**
+ * Holds the content for converting user data to json format and also used for
+ * holding content from json file
+ * 
+ * @author mjq34833
+ *
+ */
+public class ParametersConverter {
+
+	String version;
```

Comment:
What is this a version of? Can we make it clearer in the code.

Reply:
Done.